classes/07class/exercises/c07-serverless02/alanlima/README.md

```diff
@@ -0,0 +1,299 @@
+  PublicDnsName: ''
+  RootDeviceName: /dev/xvda
+  RootDeviceType: ebs
+  SecurityGroups:
+  - GroupId: sg-df6655aa
+    GroupName: default
+  SourceDestCheck: true
+  State:
+    Code: 0
+    Name: pending
+  StateReason:
+    Code: pending
+    Message: pending
+  StateTransitionReason: ''
+  SubnetId: subnet-957b9bdd
+  Tags:
+  - Key: untagged-tracker
+    Value: 'true'
+  VirtualizationType: hvm
+  VpcId: vpc-fc020c9b
+OwnerId: '097922957316'
+ReservationId: r-09d1f1a377bbbc575
+```
+
+### Running an instance **with** `CostCentre` tag
+
+```bash
+$ make run-instance-with-tag
+Groups: []
+Instances:
+- AmiLaunchIndex: 0
+  Architecture: x86_64
+  BlockDeviceMappings: []
+  CapacityReservationSpecification:
+    CapacityReservationPreference: open
+  ClientToken: f9dc9292-8ad3-4a02-a4df-d06ddeff62b7
+  CpuOptions:
+    CoreCount: 1
+    ThreadsPerCore: 1
+  EbsOptimized: false
+  Hypervisor: xen
+  ImageId: ami-04025138b31328de6
+  InstanceId: i-0d42e836480e38abd
+  InstanceType: t2.micro
+  LaunchTime: '2020-09-10T23:40:39+00:00'
+  MetadataOptions:
+    HttpEndpoint: enabled
+    HttpPutResponseHopLimit: 1
+    HttpTokens: optional
+    State: pending
+  Monitoring:
+    State: disabled
+  NetworkInterfaces:
+  - Attachment:
+      AttachTime: '2020-09-10T23:40:39+00:00'
+      AttachmentId: eni-attach-0615cf08955edd3d4
+      DeleteOnTermination: true
+      DeviceIndex: 0
+      Status: attaching
+    Description: ''
+    Groups:
+    - GroupId: sg-df6655aa
+      GroupName: default
+    InterfaceType: interface
+    Ipv6Addresses: []
+    MacAddress: 06:74:4e:e2:80:44
+    NetworkInterfaceId: eni-0249dcb2ef33f1610
+    OwnerId: '097922957316'
+    PrivateDnsName: ip-172-31-32-208.ap-southeast-2.compute.internal
+    PrivateIpAddress: 172.31.32.208
+    PrivateIpAddresses:
+    - Primary: true
+      PrivateDnsName: ip-172-31-32-208.ap-southeast-2.compute.internal
+      PrivateIpAddress: 172.31.32.208
+    SourceDestCheck: true
+    Status: in-use
+    SubnetId: subnet-957b9bdd
+    VpcId: vpc-fc020c9b
+  Placement:
+    AvailabilityZone: ap-southeast-2a
+    GroupName: ''
+    Tenancy: default
+  PrivateDnsName: ip-172-31-32-208.ap-southeast-2.compute.internal
+  PrivateIpAddress: 172.31.32.208
+  ProductCodes: []
+  PublicDnsName: ''
+  RootDeviceName: /dev/xvda
+  RootDeviceType: ebs
+  SecurityGroups:
+  - GroupId: sg-df6655aa
+    GroupName: default
+  SourceDestCheck: true
+  State:
+    Code: 0
+    Name: pending
+  StateReason:
+    Code: pending
+    Message: pending
+  StateTransitionReason: ''
+  SubnetId: subnet-957b9bdd
+  Tags:
+  - Key: CostCentre
+    Value: dev
+  - Key: untagged-tracker
+    Value: 'true'
+  VirtualizationType: hvm
+  VpcId: vpc-fc020c9b
+OwnerId: '097922957316'
+ReservationId: r-0d3139696da4a2240
+```
+
+### Listing created instances
+
+```bash
+$ make list-instances
+- Instance: i-0d42e836480e38abd
+  State: running
+  Tags:
+  - Key: CostCentre
+    Value: dev
+  - Key: untagged-tracker
```

Comment:
is this `untagged-tracker` a way of knowing if the lambda has checked this instance?